canonical-web-and-design/juju.is
# Add hero logo padding on small screens

## Done

- Delete redundant tabpanel scss/js
- Add hero image padding on small screens

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8041
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Verify that logos in hero have increased padding on small screens


## Issue / Card

Fixes #142

## Screenshots

<img width="492" alt="Screenshot 2020-10-29 at 08 57 15" src="https://user-images.githubusercontent.com/505570/97546750-c5568c00-19c4-11eb-9bcd-9f8e7a589817.png">
